bofire/domain/domain.py

```diff
@@ -42,9 +54,20 @@ class Domain(BaseModel):
         constraints (List[Constraint], optional): List of constraints. Defaults to [].
     """
 
+    def outputs(self) -> OutputFeatures:
```

Comment:
This change is fine for me, but I would prefer to set it up as property using the @property decorator so that it behaves like an attribute. What do you think?

Reply:
Yes, we can do that.

Reply:
Do you do it?

Reply:
Yes. I shouldn't have used the confusing "we". :)